frappe/recorder.py

```diff
@@ -252,9 +253,12 @@ def process_profiler(self):
 		if self.config.profile or self.profiler:
 			self.profiler.disable()
 			profiler_output = io.StringIO()
-			pstats.Stats(self.profiler, stream=profiler_output).strip_dirs().sort_stats(
-				"cumulative"
-			).print_stats(200)
+			if self.config.dir_output:
+				pstats.Stats(self.profiler, stream=profiler_output).sort_stats("cumulative").print_stats()
+			else:
+				pstats.Stats(self.profiler, stream=profiler_output).strip_dirs().sort_stats(
```

Comment:
IMO No need to make this complicated. We can remove `strip_dirs` for everyone. I only added it because it's somewhat easier to read if you already know the code, but I guess it slows down the process when you're profiling unknown code.

Reply:
Hi @ankush, I've followed your instruction by simply remove `strip_dirs`